src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -263,6 +266,9 @@ public void run() {
     }
 
     if (newVar != null) {
+      if (intoScalar) {
```

Comment:
Shouldn't you verify here that `type` is a collection?

Reply:
Done